resolwe/flow/managers/workload_connectors/kubernetes.py

```diff
@@ -0,0 +1,603 @@
+            # specified by the --root-dir flag.
+            # https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+            #
+            # TODO: how will this policy be transfered to kubelets? Currently I mount
+            # my /tmp directory to the /seccomp directory in minikube.
+
+            # TODO: enable when sollution is found.
+            # security_context["seccompProfile"] = {
+            #    "type": "Localhost",
+            #    "localhostProfile": "test",
+            # }
+            pass
+
+        communicator_image = getattr(
+            settings, "DOCKER_COMMUNICATOR_IMAGE", "resolwe/com:python-3.8"
+        )
+
+        # HACK: make nicer.
+        requirements = data.process.requirements.get("executor", {}).get("docker", {})
+
+        job_description = {
+            "apiVersion": "batch/v1",
+            "kind": "Job",
+            "metadata": {"name": job_name},
+            "spec": {
+                "ttlSecondsAfterFinished": 6000,  # Must enable feature to work (minikube start --mount --mount-string="/home/gregor:/home/gregor" --feature-gates="TTLAfterFinished=true")
+                "template": {
+                    "metadata": {"name": job_name},
+                    "spec": {
+                        "hostNetwork": use_host_network,
+                        "volumes": self._volumes(
+                            secrets_name,
+                            configmap_name,
+                            efs_claim_name,
+                            location_subpath,
+                            core_api,
+                        ),
+                        "containers": [
+                            {
+                                "name": container_name,
+                                "image": requirements.get(
+                                    "image", constants.DEFAULT_CONTAINER_IMAGE
+                                ),
+                                "resources": {"limits": limits, "requests": requests},
+                                "securityContext": security_context,
+                                "env": self._prepare_environment(data),
+                                "workingDir": constants.DATA_VOLUME,
+                                "imagePullPolicy": "Always",
+                                "command": ["/usr/bin/python3"],
+                                "args": ["/processing.py"],
+                                "volumeMounts": self._processing_mountpoints(
+                                    location_subpath
+                                ),
+                            },
+                            {
+                                "name": f"{container_name}-communicator",
+                                "image": communicator_image,
+                                "imagePullPolicy": "Always",
+                                "resources": {
+                                    "limits": {"cpu": 0.5, "memory": "1024M"},
```

Comment:
How about this settings:
`                                "resources": {
                                    "limits": {"cpu": 0.5, "memory": "1024M"},
                                    "requests": {"memory": "256M"},
                                },
`
This will schedule the pod on any node that has 256MB RAM available for com container.
But it will only be killed when consuming more than 1G of RAM. The memory usage can spike at the file collection phase if there are a lot of files to collect. However, the processing container is already stopped by then and there should be plenty of memory available.